DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: googleAuthR
 Type: Package
-Version: 0.8.1.9000
+Version: 0.8.2.9000
```

Comment:
This would be `0.8.1.9001` - `9000` is the dev versioning not on CRAN, once on CRAN it would be `0.8.2`

Reply:
Thanks. I'll also peruse through the docs again, I think I saw an inconsistency in there somewhere...